inc/saiswitch.h

```diff
@@ -490,6 +495,32 @@ typedef enum _sai_switch_tunnel_attr_t
      */
     SAI_SWITCH_TUNNEL_ATTR_VXLAN_UDP_SPORT,
 
+    /**
+     * @brief Tunnel UDP source port base for a calculated hash range
+     *
+     * Value of the fixed portion of the UDP source port's MSB
```

Comment:
perhaps would be more clear to write most significant bits in plural as MSB is most significant bit
same for LSB below